docs/design/Spacing.stories.mdx

```diff
@@ -39,6 +39,14 @@ reflect their relation to each other. An example would be the spacing between
 
 ![illustration of a small space between Chip components](https://user-images.githubusercontent.com/39704901/143141613-193ceeb2-b393-44ab-a5e8-21d2f88c67c4.png)
 
+### Slim
+
+Use `slim` spacing when you don't quite need a `small` space, but `base` is just
+too much. This is often useful for optically balancing the internal padding of a
+UI element.
+
+![illustration of a slim space between two components](https://user-images.githubusercontent.com/39704901/143141629-77777777-7777-7777-7777-777777777777.png)
```

Comment:
TODO: get image

Going to replace all of the imagery in this doc since it's all outdated

Reply:
![image](https://github.com/user-attachments/assets/f9b6e909-8866-4a0f-acf3-8eb7be4e703c)

Reply:
Done - replaced with Figma embeds